meta-nemo/recipes-core/psplash/psplash_git.bbappend

```diff
@@ -0,0 +1 @@
+SPLASH_IMAGES = "file://psplash-img-320.png;outsuffix=default"
```

Comment:
You can change this line to either: 
- `psplash-img-280-154.gif`
- `psplash-img-320-176.gif`
- `psplash-img-400-220.gif`

This might make it a little less obvious but it will still show a smaller bootlogo as the resolution doesn't match any of our current gifs.

Reply:
Since *-400-220.gif is ment for 400px² screens. I think we can safely assume we need a new `psplash-img-480-264.gif` to fir with the transition animation to homescreen. But changing to the 400px one for the time being.